maps/src/main/java/org/odk/collect/maps/layers/OfflineMapLayersPickerViewModel.kt

```diff
@@ -0,0 +1,38 @@
+package org.odk.collect.maps.layers
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.odk.collect.async.Scheduler
+import org.odk.collect.settings.SettingsProvider
+import org.odk.collect.settings.keys.ProjectKeys
+
+class OfflineMapLayersPickerViewModel(
+    private val referenceLayerRepository: ReferenceLayerRepository,
+    scheduler: Scheduler,
+    private val settingsProvider: SettingsProvider
+) : ViewModel() {
+    private val _data = MutableLiveData<Pair<List<ReferenceLayer>, String?>>()
+    val data: LiveData<Pair<List<ReferenceLayer>, String?>> = _data
```

Comment:
I agree and I've come to the same conclusion but at a later stage (while working on deleting layers when I need even more details like for example if the layer item is expanded or collapsed). If it's ok, let's maybe keep it unchanged for now so that I can cherry-pick commits that will improve this code without any conflicts (in one of the next prs).

Reply:
Done.